### PR TITLE
CTCP-4474: Update sass

### PR DESCRIPTION
**Pull request by justega247**
* https://github.com/hmrc/play-frontend-hmrc/blob/3cbd82d8ad2952b6310867d25eb5cc171bc16cae/docs/maintainers/sass-compilation.md
* application-ie8.scss removed ahead of govuk-frontend 5.0.0 release
* Removing unused ctc-add-to-a-list styling

Screenshot 2023-11-29 at 13 34 09

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-@import "lib/govuk-frontend/govuk/all";
-@import "lib/hmrc-frontend/hmrc/all";
+$govuk-include-default-font-face: false;
+@import "lib/govuk-frontend/govuk/base";
 
 @import "partials/shame";
 
 .summary-title {
-  @extend .govuk-heading-m;
+  @extend .govuk-heading-m !optional;
```
Comment (joecefnhedog) [inline app/assets/stylesheets/application.scss:7]: Don't know if this should be optional, or keep the relevant imports

Comment (justega247) [reply app/assets/stylesheets/application.scss:7]: Keeping it as it was stops the code from compiling.

Comment (joecefnhedog) [reply app/assets/stylesheets/application.scss:7]: I think this one will fix it, 
`@import "lib/govuk-frontend/govuk/all";`

Comment (justega247) [reply app/assets/stylesheets/application.scss:7]: That is one of the one we are meant to replace from the ticket description.

Comment (joecefnhedog) [reply app/assets/stylesheets/application.scss:7]: Yeah, may be worth confirming with Jack. I read it that we should remove  - but double check there are no errors

Comment (jackheslop96) [reply app/assets/stylesheets/application.scss:7]: Ah interesting. Yes suspect we can keep the `@import "lib/govuk-frontend/govuk/all";` in that case

**Review by karahanozturk — APPROVED** [review]

**Pull request merged**